paddle/fluid/inference/api/mkldnn_quantizer.cc

```diff
@@ -131,7 +131,8 @@ void AnalysisPredictor::MkldnnQuantizer::CalculateScalesForOpOutputs(
         is_unsigned = true;
       } else if (op->Type() == "transpose2" || op->Type() == "reshape2" ||
                  op->Type() == "pool2d" || op->Type() == "nearest_interp" ||
-                 op->Type() == "nearest_interp_v2" || op->Type() == "split") {
+                 op->Type() == "nearest_interp_v2" || op->Type() == "split" ||
+                 "softmax") {
```

Comment:
This section is for immutable ops which causes scale for output to be copied from scale for input (line 143).

Reply:
So where softmax should be put in your opinion?

Reply:
You could skip adding softmax here entirely if you want the default behavior of scale calculation for output (which would require change of `rules_["softmax"]["Out"] = ScaleAlgo::NONE;` to some method).